src/mocks/handlers/dictionary.handlers.ts

```diff
@@ -115,6 +106,18 @@ export const handlers = [
 		return res(ctx.status(200), ctx.json(saved));
 	}),
 
+	rest.put('/umbraco/management/api/v1/dictionary/:id', async (req, res, ctx) => {
```

Comment:
Please use the function `umbracoPath` for any mock URLs. Check the other handlers to see how it is used.